int8 shoulder
int8 upperArm
int8 midArm
int8 lowerArm
int8 wrist
int8 finger
